test/integration/account_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
 // +build integration
```

Comment:
I guess `+build integration` can be removed as `go:build integration` is now preferred

Reply:
OK, I'll change this (I've added this because go ver wrote that this syntax is better)

Reply:
@ndrewnee Should I change assert to require in other files?